datastore/blob/azure_storage_api/__init__.py

```diff
@@ -165,7 +165,7 @@ async def is_a_folder(container_client, folder_name):
         raise Exception("Datastore.blob.azure_storage : Unhandled Error")
 
 
-async def create_folder(container_client, folder_uuid=None, folder_name=None):
+async def create_folder(container_client, folder_uuid=None, folder_name=None, file_name=None):
```

Comment:
I think you forgot to update the documentation with the new parameter file name (I actually was wondering what it was)

Reply:
Yeah thats quite hard to explain. I had an issue because the default blob name for a new folder was just `folder_name/folder_name.json` in the container but as we wanted to have a folder by user in the dev container i needed a custom blob name which is `user_id/folder_name/folder_name.json` to add the deleted picture_set in it. Maybe i should rename the parameter blob_name and a description would be "customize blob name" or if you have a better idea ?

Reply:
No, just add the parameter in the list of parameter with a simple description. Or you could rename it index_name, since those json are used as Indexes in the storage (As we are moving on, I really wonder the utility of those indexes...)